packages/cli/src/commands/setup/tailwind/tailwind.js

```diff
@@ -191,5 +191,6 @@ export const handler = async ({ force }) => {
     await tasks.run()
   } catch (e) {
     console.log(c.error(e.message))
```

Comment:
Use stderr, instead of stdout.
```suggestion
    console.error(c.error(e.message))
```